src/experimental-layers/src/path-marker-layer/create-path-markers.js

```diff
@@ -72,9 +74,20 @@ function createMarkerAlongPath({path, percentage, lineLength, color, object, vie
     .multiply(new Vector2(along, along))
     .add(path[i]);
 
-  const vDirection2 = new Vector2(viewport.projectFlat(path[i + 1]))
-    .subtract(viewport.projectFlat(path[i]));
+  const projectFlat = coordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS ?
```

Comment:
This is a little too compact for my taste. Since we are executing statements and not evaluating expressions this should be a proper if statement instead of a ternary (and maybe wrap it in a real function rather than than trying to save an if statement?). If a real function we can think about how to move it "upstream".

Also you may want to `npm run lint` to fix the spacing issues etc. pretty sure that linter is failing in CI.